project/Dependencies.scala

```diff
@@ -7,27 +7,30 @@ object Dependencies {
 
   val scala210 = "2.10.7"
   val scala211 = "2.11.12"
-  val scala212 = "2.12.15"
-  val scala213 = "2.13.12"
+  val scala212 = "2.12.19"
+  val scala213 = "2.13.13"
   val scala3 = "3.3.1"
+  val scala3ForBridge = "3.3.1"
+  val scala213ForBridge = "2.13.13"
   val defaultScalaVersion = scala3
-  val allScalaVersions = Seq(scala212, scala210, scala211, scala213)
+  val compilerBridgeVersions = Seq(scala212, scala210, scala211, scala213)
   val scala212_213 = Seq(defaultScalaVersion, scala213)
   val scala3_only = Seq(scala3)
 
-  private val ioVersion = nightlyVersion.getOrElse("1.9.7")
-  private val utilVersion = nightlyVersion.getOrElse("2.0.0-alpha8")
+  private val ioVersion = nightlyVersion.getOrElse("1.10.0")
+  private val utilOrganization = "ch.epfl.scala"
+  private val utilVersion = nightlyVersion.getOrElse("2.0.0-alpha11")
```

Comment:
I published the utils of https://github.com/sbt/sbt/pull/7554 to the `ch.epfl.scala` org and version `2.0.0-alpha11`

Reply:
alpha10 is already from the branch  btw.

Reply:
Oh yes you are right. I don't know why I thought we would also need `collections` which is missing on `2.0.0-alpha10`. But we don't.